app/core/models.py

```diff
@@ -62,26 +105,37 @@ class User(PermissionsMixin, AbstractBaseUser, AbstractBaseModel):
 
     # Django-user related fields #
     # password is inherited from AbstractBaseUser
-    email = models.EmailField("Email address", blank=True)  # allow non-unique emails
+    email_intake = models.EmailField(
+        "Email address", blank=True
+    )  # allow non-unique emails
     is_staff = models.BooleanField(
         "staff status",
         default=False,
         help_text="Designates whether the user can log into this admin site.",
     )
 
-    first_name = models.CharField(max_length=255, blank=True)
-    last_name = models.CharField(max_length=255, blank=True)
-    gmail = models.EmailField(blank=True)
-    preferred_email = models.EmailField(blank=True)
+    name_first = models.CharField(max_length=255, blank=True)
+    name_last = models.CharField(max_length=255, blank=True)
+    email_gmail = models.EmailField(blank=True)
+    email_preferred = models.EmailField(blank=True)
+    email_cognito = models.EmailField(blank=True)
 
     user_status = models.ForeignKey(
         "UserStatusType", null=True, on_delete=models.PROTECT
     )
     # current_practice_area = models.ManyToManyField("PracticeArea")
-    # target_practice_area = models.ManyToManyField("PracticeArea")
+    practice_area_primary = models.ManyToManyField(
```

Comment:
I think this field is just a FK relation rather than many-to-many.

Also, you should delete the "current_practice_area" comment above as well.